de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/Desktop.gss

```diff
@@ -168,7 +172,7 @@
     gwt-sprite: "dataBtn";
     height: WIN_BTN_HEIGHT !important;
     width: WIN_BTN_WIDTH !important;
-    margin-top: WIN_BTN_MARGIN_TOP;
```

Comment:
What's this change? I guess the margin needed to be bigger given some sort of margin collapsing, or?